src/main/java/com/alkemy/ong/security/ApiSecurityConfig.java

```diff
@@ -45,7 +45,7 @@ public void configure(HttpSecurity http) throws Exception {
         http.addFilter(customAuthenticationFilter);
         http.addFilterBefore( customAuthorizationFilter, CustomAuthenticationFilter.class );
         http.authorizeRequests().antMatchers(LOGIN_URL,REGISTER_URL).permitAll();
-//        http.authorizeRequests().antMatchers(SWAGGER_URL).permitAll(); //Descomententar esta linea para usar swagger
+        http.authorizeRequests().antMatchers(SWAGGER_URL).permitAll();
```

Comment:
No entiendo esta línea, si agregas para jwt, porque agregas que todos puedan acceder? Incluso no autenticados. Si pudieras Responderme esa inquietud, te lo agradezco.

Reply:
En /api/docs pueden entrar todos pero para utilizarla deben tener el token. Estaría bueno poder proteger swagger antes de entrar pero los tutoriales que vi hacian uso del front con una pagina para el login. Para que solo puedan ingresar los ADMIN solo se tendría que permitir el acceso al JSON para usarlo sin la interfaz grafica. Se me hace bastante dificil explicar por aca pero mañana despues de la planning hacemos un meet y veo si mostrando me doy a entender mejor

Reply:
De acuerdo. Igualmente solo quería entenderlo mejor. Porque yo creía que no hacía falta porque te pedía "autorizar" Porque yo vi esto: https://stackoverflow.com/questions/59898874/enable-authorize-button-in-springdoc-openapi-ui-for-bearer-token-authentication